pandas/tests/test_base.py

```diff
@@ -1096,6 +1096,21 @@ def f():
         self.assertFalse(hasattr(t, "b"))
 
 
+class TestTile(tm.TestCase):
+
+    def test_single_bin(self):
+
+        expected = pd.Series([0, 0])
+
+        s = pd.Series([9., 9.])
```

Comment:
pandas\tools\tests\test_tile.py

lots of tests there, just add this one on., no need to create a new class. add a comment with the issue number